tensorbay/opendataset/SegTrack2/loader.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+#
+# Copytright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+# pylint: disable=missing-module-docstring
+
+import os
+from typing import Callable
+
+import numpy as np
+
+from ...dataset import Data, Dataset
+from ...label import InstanceMask
+from .._utility import glob
+
+try:
+    from PIL import Image
+except ModuleNotFoundError:
+    from .._utility.mocker import Image
+
+DATASET_NAME = "SegTrack2"
+
+_SEGMENTS_INFO = {
```

Comment:
In continuous datasets, we need to make sure every segment is continuous, which means one image sequence for one segment, we do not put two or more image sequences in one segment which break the continuity of the segment.
Seems there are more than one sequence per categories in this dataset.